src/main/kotlin/app/revanced/patches/remini/misc/premium/fingerprints/UnlockPremiumFingerprint.kt

```diff
@@ -0,0 +1,19 @@
+package app.revanced.patches.remini.misc.premium.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.Opcode
+
+object UnlockPremiumFingerprint : MethodFingerprint(
+    opcodes = listOf(
+        Opcode.IGET_BOOLEAN,
+        Opcode.IF_NEZ,
+        Opcode.INVOKE_VIRTUAL,
+        Opcode.MOVE_RESULT_OBJECT,
+        Opcode.IGET_BOOLEAN,
+        Opcode.IF_EQZ,
+        Opcode.GOTO,
+        Opcode.CONST_4
+    ),
+    customFingerprint = { it.parameterTypes[0] == "Ljava/lang/Object;" },
```

Comment:
This will fail with `IndexOutOfBoundsException` if the method has no parameters. Check on the length.